sfs/util.py

```diff
@@ -418,32 +418,57 @@ def apply(self, func, *args, **kwargs):
 """
 
 
-def image_sources_for_box(x, L, max_order, strict_order=True):
-    """Image source method for cuboid room.
+def image_sources_for_box(x, L, N, prune=True):
+    """Image source method for a cuboid room.
+
+    The classical method by Allen & Berkley [1].
```

Comment:
I get this error:

    .../sfs/util.py:docstring of sfs.util.image_sources_for_box:26: WARNING: Footnote [1] is not referenced.

Is there some extension needed?

Reply:
Turns out I was using an older version of Sphinx where this was fine.
Updated.